nautobot/extras/templatetags/job_buttons.py

```diff
@@ -28,7 +28,7 @@
 <input type="hidden" name="object_pk" value="{object_pk}">
 <input type="hidden" name="object_model_name" value="{object_model_name}">
 <input type="hidden" name="_schedule_type" value="immediately">
-<input type="hidden" name="_task_queue" value="{task_queue}">
+<input type="hidden" name="_job_queue" value="{job_queue}">
```

Comment:
Does this work correctly passing a queue name instead of a PK? Because at a glance it looks like the updated view code is expecting a PK.

Reply:
Manually tested job buttons and it's working.